### PR TITLE
New package: font-ibm-plex-1.0.2

### PR DESCRIPTION
**Pull request by Vintodrimmer**
New package: font-ibm-plex-otf-1.0.0
New package: font-ibm-plex-ttf-1.0.0

### CODE REVIEW

File: srcpkgs/font-ibm-plex-ttf/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'font-ibm-plex-ttf'
+pkgname=font-ibm-plex-ttf
+version=1.0.2
+revision=1
+wrksrc=plex-${version}
+noarch="yes"
```
Comment (maxice8) [inline srcpkgs/font-ibm-plex-ttf/template]: unquote noarch, quote wrksrc

File: srcpkgs/font-ibm-plex-otf/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'font-ibm-plex-otf'
+pkgname=font-ibm-plex-otf
+version=1.0.2
+revision=1
+wrksrc=plex-${version}
+noarch="yes"
```
Comment (maxice8) [inline srcpkgs/font-ibm-plex-otf/template]: unquote noarch, quote wrksrc

**Comment by Vintodrimmer** [comment]
Should I set new revision for such changes or does it make sense only once the package is in the repo so it gets updated?

**Comment by maxice8** [comment]
you're introducing a new package, one commit per package introduced. don't even bother making corrections just git --amend them.

**Comment by maxice8** [comment]
i mean don't bother making new commits to correct stuff, just use git commit --amend to add the changes to the commit. if you need to go to a previous commit you can do git rebase -i master.

**Comment by Vintodrimmer** [comment]
Last time I failed a bit, but I'll try!

**Comment by Vintodrimmer** [comment]
I"ve made a new one for the new repositories [here](https://github.com/void-linux/void-packages/pull/1318), so i'm closing this one.

**Pull request closed without merging**